espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/ViewAttributesEnum.java

```diff
@@ -0,0 +1,34 @@
+package io.appium.espressoserver.lib.model;
+
+public enum ViewAttributesEnum {
+
+    CONTENT_DESC("content-desc"),
+    CLASS("class"),
+    TEXT("text"),
+    PACKAGE("package"),
+    CHECKABLE("checkable"),
+    CHECKED("checked"),
+    CLICKABLE("clickable"),
+    ENABLED("enabled"),
+    FOCUSABLE("focusable"),
+    FOCUSED("focused"),
+    SCROLLABLE("scrollable"),
+    LONG_CLICKABLE("long-clickable"),
+    PASSWORD("password"),
+    SELECTED("selected"),
+    BOUNDS("bounds"),
+    RESOURCE_ID("resource-id"),
+    INSTANCE("instance"),
+    INDEX("index");
+
+
+    private final String name;
```

Comment:
each enum element already has **name** property. We could simply transform it to lowecase, replace underscore with dash and override **toString** to avoid having extra constructor.